tests/e2e/util.go

```diff
@@ -5325,7 +5325,7 @@ func execDockerPauseNKillOnContainer(sshClientConfig *ssh.ClientConfig, k8sMaste
 		k8sMasterNodeIP)
 	cmdResult, err := sshExec(sshClientConfig, k8sMasterNodeIP,
 		containerPauseCmd)
-	if err != nil || cmdResult.Code != 0 {
+	if err != nil && cmdResult.Code != 0 {
```

Comment:
why are we changing this now?

Reply:
While running nerdctl kill command, we are getting  warnings which it is considering as an error, but the kill command is getting executed successfully.

cmdResult.Code always returns 0, but due to OR condition, err != nil || cmdResult.Code != 0, the code was failing

Hence, updated OR condition with AND condition that if both condition satisfies only then it should fail the code.

Reply:
may be you can check for specific error string and then continue if it matches

Reply:
This check we are already making in the code. Removing AND condition as it is not required.